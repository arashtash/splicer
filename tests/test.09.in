molestie ut. Fusce sodales varius enim sed sagittis. EOS
varius purus dapibus eu. Vestibulum eget nunc ac EOS
per inceptos himenaeos. Proin et lorem convallis, finibus EOS
consectetur adipiscing elit. Pellentesque sed dolor non arcu EOS
libero tincidunt sed. Aliquam felis sapien, fringilla at varius nec, EOS
fringilla lacus, tristique laoreet ligula. Pellentesque a sagittis EOS
porttitor iaculis. Nulla facilisi. Nam nec neque sed neque viverra EOS
ac tincidunt eget, commodo at nibh. Integer a fringilla EOS
sed sagittis. Donec interdum efficitur eros, in rutrum quam EOS
Nulla fringilla rhoncus ex, vitae ultrices urna molestie ut. EOS
pellentesque dapibus. Vivamus tincidunt fringilla nunc, vel sollicitudin arcu EOS
ultrices urna molestie ut. Fusce sodales varius enim EOS
ultrices. Nunc ante ex, cursus non ultrices at, interdum vitae EOS
accumsan ante, ut facilisis turpis felis in urna. In viverra EOS
aliquet ipsum. Proin vestibulum sollicitudin erat eu molestie. Maecenas rutrum, EOS
sit amet tellus mauris. Duis a lectus ligula. Nam viverra EOS
congue id quis felis. Integer ultricies risus et EOS
Etiam nibh tortor, posuere ac tincidunt eget, commodo EOS
congue sit amet pulvinar ac, imperdiet et purus. Aenean EOS
felis. Integer ultricies risus et viverra fermentum. Nam et metus EOS
at, vulputate metus. Phasellus vitae egestas felis. Proin a aliquet EOS
sapien, fringilla at varius nec, malesuada vel arcu. EOS
fermentum. Nam et metus dignissim, gravida lacus blandit, sodales EOS
urna vel magna pharetra, et placerat libero ultrices. Nunc ante EOS
Fusce sodales varius enim sed sagittis. Donec interdum EOS
dignissim, gravida lacus blandit, sodales est. Duis tincidunt, EOS
Etiam dictum, ligula id luctus egestas, massa mi congue odio, EOS
id augue. Morbi urna mauris, auctor nec tortor EOS
nibh. Integer a fringilla lacus, tristique laoreet ligula. EOS
eu molestie. Maecenas rutrum, ligula pellentesque sodales vestibulum, est EOS
eu. Vestibulum eget nunc ac ligula dictum feugiat id at EOS
ac, imperdiet et purus. Aenean quis aliquet ipsum. Proin vestibulum EOS
luctus nisl consectetur vel. Fusce auctor a orci sit EOS
ad litora torquent per conubia nostra, per inceptos himenaeos. Proin EOS
tincidunt fringilla nunc, vel sollicitudin arcu convallis elementum. EOS
quam est, suscipit elementum ante porttitor iaculis. Nulla facilisi. EOS
et placerat dignissim. Fusce sit amet tellus mauris. Duis a EOS
viverra sem eros, nec varius purus dapibus eu. Vestibulum EOS
et placerat libero ultrices. Nunc ante ex, cursus non EOS
odio, eget dapibus ipsum nisi mattis mi. Aliquam EOS
et lorem convallis, finibus ipsum at, vulputate metus. Phasellus EOS
sodales est. Duis tincidunt, turpis eu posuere pellentesque, urna nisi EOS
vel. Fusce auctor a orci sit amet cursus. Quisque lacinia EOS
egestas, massa mi congue odio, eget dapibus ipsum nisi EOS
non arcu imperdiet placerat. Pellentesque ullamcorper urna vel magna EOS
sollicitudin arcu convallis elementum. Nullam facilisis sit amet lacus sed EOS
eget convallis diam. Ut dapibus tellus vel nisi maximus, sit EOS
felis. Proin a aliquet felis, eget convallis diam. Ut EOS
mi. Aliquam augue metus, congue sit amet pulvinar EOS
ultrices at, interdum vitae libero. Etiam nibh tortor, posuere ac EOS
amet pharetra velit vestibulum. Etiam dictum, ligula id luctus egestas, EOS
tortor ut, vestibulum ornare nisi. Nulla fringilla rhoncus ex, EOS
vel arcu. Fusce auctor mi et placerat dignissim. Fusce EOS
turpis eu posuere pellentesque, urna nisi accumsan ante, ut facilisis EOS
Curabitur molestie pharetra arcu, a rutrum libero tincidunt sed. Aliquam EOS
nisi maximus, sit amet pharetra velit vestibulum. Etiam EOS
Class aptent taciti sociosqu ad litora torquent per conubia EOS
felis in urna. In viverra sem eros, nec varius EOS
at varius nec, malesuada vel arcu. Fusce auctor mi EOS
elit. Pellentesque sed dolor non arcu imperdiet placerat. Pellentesque EOS
Phasellus ultricies eleifend leo, id luctus nisl consectetur vel. Fusce EOS
Ut aliquam leo eget nisl pellentesque dapibus. Vivamus tincidunt EOS
placerat. Pellentesque ullamcorper urna vel magna pharetra, et placerat EOS
efficitur eros, in rutrum quam ultrices dapibus. Phasellus ultricies EOS
porta leo, eget vestibulum eros tellus id augue. Morbi urna EOS
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque EOS
ligula. Pellentesque a sagittis neque. Curabitur molestie pharetra arcu, EOS
sit amet cursus. Quisque lacinia placerat pellentesque. Ut EOS
lacinia placerat pellentesque. Ut aliquam leo eget nisl pellentesque EOS
nec neque sed neque viverra congue id quis felis. EOS
Nam viverra quam est, suscipit elementum ante porttitor EOS
dictum feugiat id at ligula. Class aptent taciti sociosqu ad EOS
Nullam facilisis sit amet lacus sed consequat. EOS
urna mauris, auctor nec tortor ut, vestibulum ornare EOS
dapibus ipsum nisi mattis mi. Aliquam augue metus, EOS
a lectus ligula. Nam viverra quam est, suscipit EOS
dapibus tellus vel nisi maximus, sit amet pharetra velit EOS
rutrum quam ultrices dapibus. Phasellus ultricies eleifend leo, id EOS
pellentesque sodales vestibulum, est lorem porta leo, eget vestibulum EOS
EOS
