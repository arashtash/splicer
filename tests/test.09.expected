Lorem ipsum dolor sit amet, consectetur 
adipiscing elit. Pellentesque sed 
dolor non arcu imperdiet placerat. 
Pellentesque ullamcorper urna vel 
magna pharetra, et placerat libero 
ultrices. Nunc ante ex, cursus 
non ultrices at, interdum vitae 
libero. Etiam nibh tortor, posuere 
ac tincidunt eget, commodo at nibh. 
Integer a fringilla lacus, tristique 
laoreet ligula. Pellentesque a 
sagittis neque. Curabitur molestie 
pharetra arcu, a rutrum libero 
tincidunt sed. Aliquam felis sapien, 
fringilla at varius nec, malesuada 
vel arcu. Fusce auctor mi et placerat 
dignissim. Fusce sit amet tellus 
mauris. Duis a lectus ligula. Nam 
viverra quam est, suscipit elementum 
ante porttitor iaculis. Nulla facilisi. 
Nam nec neque sed neque viverra 
congue id quis felis. Integer ultricies 
risus et viverra fermentum. Nam 
et metus dignissim, gravida lacus 
blandit, sodales est. Duis tincidunt, 
turpis eu posuere pellentesque, 
urna nisi accumsan ante, ut facilisis 
turpis felis in urna. In viverra 
sem eros, nec varius purus dapibus 
eu. Vestibulum eget nunc ac ligula 
dictum feugiat id at ligula. Class 
aptent taciti sociosqu ad litora 
torquent per conubia nostra, per 
inceptos himenaeos. Proin et lorem 
convallis, finibus ipsum at, vulputate 
metus. Phasellus vitae egestas 
felis. Proin a aliquet felis, eget 
convallis diam. Ut dapibus tellus 
vel nisi maximus, sit amet pharetra 
velit vestibulum. Etiam dictum, 
ligula id luctus egestas, massa 
mi congue odio, eget dapibus ipsum 
nisi mattis mi. Aliquam augue metus, 
congue sit amet pulvinar ac, imperdiet 
et purus. Aenean quis aliquet ipsum. 
Proin vestibulum sollicitudin erat 
eu molestie. Maecenas rutrum, ligula 
pellentesque sodales vestibulum, 
est lorem porta leo, eget vestibulum 
eros tellus id augue. Morbi urna 
mauris, auctor nec tortor ut, vestibulum 
ornare nisi. Nulla fringilla rhoncus 
ex, vitae ultrices urna molestie 
ut. Fusce sodales varius enim sed 
sagittis. Donec interdum efficitur 
eros, in rutrum quam ultrices dapibus. 
Phasellus ultricies eleifend leo, 
id luctus nisl consectetur vel. 
Fusce auctor a orci sit amet cursus. 
Quisque lacinia placerat pellentesque. 
Ut aliquam leo eget nisl pellentesque 
dapibus. Vivamus tincidunt fringilla 
nunc, vel sollicitudin arcu convallis 
elementum. Nullam facilisis sit 
amet lacus sed consequat. 
